### PR TITLE
Changing Annotation naming to Take Notes

### PR DESCRIPTION
**Pull request by shljessie**
Changing Annotation Button name to Take Notes! 
Also changed namings within the code! 

Screen Shot 2022-08-19 at 3 09 22 PM

### CODE REVIEW

**Review by matteocargnelutti — APPROVED** [review]
LGTM 😄 ! Thanks @shljessie !

CC @lizadaly

File: web/frontend/components/TakeNotesCloner.vue [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <template>
-<div id="annotation-cloner">
-  <button class="action annotate-casebook-nodes" data-disable-with="Clone-Node" v-on:click="displayModal()">Annotate {{properType()}}</button>
+<div id="take-notes-cloner">
+  <button class="action annotate-casebook-nodes" data-disable-with="Clone-Node" v-on:click="displayModal()">Take notes in {{properType()}}</button>
```
Comment (cath9) [inline web/frontend/components/TakeNotesCloner.vue]:
```suggestion
  <button class="action annotate-casebook-nodes" data-disable-with="Clone-Node" v-on:click="displayModal()">Take notes</button>
```

File: web/frontend/components/TakeNotesCloner.vue [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
   <Modal v-if="showModal"
             @close="showModal = false">
     <template class="modal-title" slot="title">
-      <span class="annotation-icon"></span>
-      <h4>Copying {{properType()}} for Annotation</h4>
+      <span class="take-notes-icon"></span>
+      <h4>Copying {{properType()}} for Taking Notes</h4>
```
Comment (cath9) [inline web/frontend/components/TakeNotesCloner.vue]:
```suggestion
      <h4>Copy this {{properType()}} to Take Notes</h4>
```

File: web/frontend/components/TakeNotesCloner.vue [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
     </template>
-    <template class="modal-body-annotation" slot="body">
-      <p class="annotation-cloner-text"> <b> To Annotate, you need to Create a Copy.<br/><br/> Choose a book</b>  you would like  "<em>{{sectionSource}}</em>" to be copied into : </p>
-      <ul class="annotation-target-list">
+    <template class="modal-body-take-notes" slot="body">
+      <p class="take-notes-cloner-text"> <b> To Take Notes, you need to Create a Copy.<br/><br/> Choose a book</b>  you would like  "<em>{{sectionSource}}</em>" to be copied into : </p>
```
Comment (cath9) [inline web/frontend/components/TakeNotesCloner.vue]:
```suggestion
      <p class="take-notes-cloner-text"> <b> To Take Notes on a specific section or resource, you need to first add a copy to an existing book in your dashboard.<br/><br/> Choose the book</b>  where you would like  "<em>{{sectionSource}}</em>" to appear: </p>
```

**Review by cath9 — COMMENTED** [review]
Updating copy

**Pull request merged**